src/ctirs/core/mongo/documents_stix.py

```diff
@@ -303,6 +304,7 @@ def split_child_nodes_stix_2_x(self):
                 if type_ != 'x-stip-sns':
                     if 'labels' in object_:
                         Tags.append_by_object(object_)
+                        self.sns_tags = object_['labels']
```

Comment:
Because `sns_tags`  is in `report` object, we should move `self.sns_tags = object_['labels']` to under `elif type_ == 'report':`

Reply:
Change the position to `self.sns_tags = set(self.sns_tags.extend(object_['labels']))` without changing it.